instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
@smarnach I don't think `EDXAPP_SETTINGS` is used anywhere before Omar introduces it in https://github.com/edx/configuration/pull/2723, so in theory I think we could set it now, and it will just be ignored until that PR gets merged. It probably makes more sense to wait until Omar's PRs get merged, though.

Even when using `EDXAPP_SETTINGS: openstack`, I believe `EDXAPP_DEFAULT_FILE_STORAGE` still defaults to "django.core.files.storage.FileSystemStorage"`, so we'll have to set it explicitly: https://github.com/edx/edx-platform/pull/11286/files#diff-8126d0664ba3411882f0ceac7d41e2e5R23